src/main.js

```diff
@@ -127,6 +133,14 @@ class MiniGraphCard extends LitElement {
     }
   }
 
+  static getStubConfig() {
+    return {
+      entities: [],
+      color_thresholds: [],
+      show: DEFAULT_SHOW,
```

Comment:
I do not understand, when this is used. If it automatically adds the default show options to each new card, we should not do this, I think. If it just can be used by the user to get a sensible template to fill in, I think it is a good idea to return all these otions here...

Reply:
This was needed because things like adding new entity on a new card was now working properly but I sorted it out internally so it is not needed anymore and will be deleted.